tests/Fixtures/e2e/Parameters_Coverage/composer.json

```diff
@@ -11,5 +11,11 @@
         "psr-4": {
             "ParamCoverage\\Test\\": "tests/"
         }
+    },
+    "config": {
```

Comment:
Is this related to the mac os builds? or is it something else?

Reply:
Nah, that's just a random bug not worthy a separate PR: test dependencies are installed just once, and without an explicit platform definition if the first install happens not from PHP 7.0, all future tests on earlier versions will go broke.